bazel/grpc_deps.bzl

```diff
@@ -263,12 +263,12 @@ def grpc_deps():
     if "com_google_protobuf" not in native.existing_rules():
         http_archive(
             name = "com_google_protobuf",
-            sha256 = "70f480fe9cb0c6829dbf6be3c388103313aacb65de667b86d981bbc9eaedb905",
-            strip_prefix = "protobuf-7f94235e552599141950d7a4a3eaf93bc87d1b22",
+            sha256 = "e9445c2188aae7dd6029a5793c01834ed3935140e7e8be44c4c12476b209cd1e",
+            strip_prefix = "protobuf-999744ca08774b54f9732880eefb46739a35c67b",
             urls = [
-                # https://github.com/protocolbuffers/protobuf/commits/v25.0
-                "https://storage.googleapis.com/grpc-bazel-mirror/github.com/protocolbuffers/protobuf/archive/7f94235e552599141950d7a4a3eaf93bc87d1b22.tar.gz",
-                "https://github.com/protocolbuffers/protobuf/archive/7f94235e552599141950d7a4a3eaf93bc87d1b22.tar.gz",
+                # https://github.com/protocolbuffers/protobuf/commits/v26.0
+                "https://storage.googleapis.com/grpc-bazel-mirror/github.com/protocolbuffers/protobuf/archive/999744ca08774b54f9732880eefb46739a35c67b.tar.gz",
```

Comment:
Need to be newer than https://github.com/protocolbuffers/protobuf/blob/32fe5482a1cc6f683256ea4944abc63036fc0733/build_defs/BUILD.bazel

Reply:
Isn't your recent PR enough to make the binder test pass? gRPC needs to use protobuf v26.0 (not main branch).

Reply:
I guess gRPC still needs https://github.com/protocolbuffers/protobuf/pull/16127 ?

Reply:
Unfortunately I think the change introduced as part of v26.0 in https://github.com/protocolbuffers/protobuf/pull/15298 is wrong and there is no way we can match it. See https://github.com/protocolbuffers/protobuf/pull/16246

Can we patch either https://github.com/protocolbuffers/protobuf/commit/e54174728f358c51a3a27f25fe275aad78828779 or https://github.com/protocolbuffers/protobuf/pull/16246 to `patches = [ ... ] `in `bazel/grpc_deps.bzl`?

Reply:
I see. Before making a patch here, I'd like to make these changes for Protobuf v26.1 first. Maintaining those patch is quite brittle, preventing users from choosing a different version of protobuf. Then would you mind helping protobuf team to get right changes on their v26.x branch?

Reply:
This is resolved.